packages/nimble-components/src/rich-text/models/tests/markdown-parser.spec.ts

```diff
@@ -534,70 +553,56 @@ describe('Markdown parser', () => {
                     {
                         name: '<bitcoin:1Hf1KqNPZzkFJ5Wv8VPop9uaF5RjKN3N9s?amount=0.001>'
                     },
-                    { name: '<javascript:vbscript:alert("not alert")>' },
                     { name: '<test://test.com>' }
-                ];
-
-                const focused: string[] = [];
-                const disabled: string[] = [];
-                for (const value of differentProtocolLinks) {
-                    const specType = getSpecTypeByNamedList(
-                        value,
-                        focused,
-                        disabled
-                    );
-                    specType(
-                        `string "${value.name}" renders as plain text within paragraph tag`,
-                        // eslint-disable-next-line @typescript-eslint/no-loop-func
+                ] as const;
+                parameterizeNamedList(differentProtocolLinks, (spec, name) => {
+                    spec(
+                        `string "${name}" renders within nimble-anchor without 'href' attribute`,
                         () => {
-                            const doc = RichTextMarkdownParser.parseMarkdownToDOM(
-                                value.name
-                            );
+                            const doc = RichTextMarkdownParser.parseMarkdownToDOM(name);
 
-                            expect(getTagsFromElement(doc)).toEqual(['P']);
-                            expect(getLeafContentsFromElement(doc)).toEqual([
-                                value.name
+                            expect(getTagsFromElement(doc)).toEqual([
+                                'P',
+                                'NIMBLE-ANCHOR'
                             ]);
+                            expect(
+                                lastChildElementHasAttribute('href', doc)
+                            ).toBeFalse();
                         }
```

Comment:
Would be good to also validate the value of `getLeafContentsFromElement(doc)`.

Reply:
Added